src/Controller/Backend/Async/UploadController.php

```diff
@@ -82,11 +82,11 @@ public function handleUpload(Request $request): JsonResponse
         });
 
         /** @var File $result */
-        $result = $uploadHandler->process($request->files->all());
+        $result = $uploadHandler->process($_FILES);
```

Comment:
undo this change, don't use globals

Reply:
I'm not sure we can.. The `process` method in the handler breaks when we pass in SF's files. It really needs the `$_FILES` array, and not an array of `Symfony\Component\HttpFoundation\File\UploadedFile`. The alternative would be to PR a fix to https://github.com/siriusphp/upload.

https://github.com/siriusphp/upload/blob/master/src/Handler.php#L196

screenshot: 

![Screenshot 2019-04-14 at 08 08 28](https://user-images.githubusercontent.com/1833361/56089031-85848e00-5e8c-11e9-987b-744adf5858e7.png)

Reply:
`$request->files->all()` does not work?

Reply:
Why do we use Sirius's upload instead of Symfony mechanisms?

Reply:
`Request->files->all` gives the error in the screenshot. 

> Why do we use Sirius's upload instead of Symfony mechanisms?

Last time i looked into it, handling them yourself in SF is a bit of a chore. Even the page itself suggests to use a third-party module: https://symfony.com/doc/current/controller/upload_file.html

I didn't like that VichUploaderBundle though, and we used this in Bolt 3 as well, and it also works well with async uploads from Vue.

Reply:
I think Symfony Community (me included) is more familiar with VichUploaderBundle than sirius. And because Sirius expects files to be $_FILES I would assume there are more places with bad habits inside their code.

Also it's really not that hard to implement our own uploading mechanism. In the back of head we need to have filesystem abstraction, like Flysystem.

Reply:
Let's add a @todo for that, then? Also, latest version of that bundle requires PHP 7.2: https://github.com/dustin10/VichUploaderBundle/blob/1.9.0/composer.json#L23

Reply:
Yeah, stuff starts requiring this version of PHP... maybe we could break with PHP 7.1?

Reply:
I am very adamant about that: We should stick with SF's requirements. We are not Marco! ;-)